src/azure-firewall/azext_firewall/custom.py

```diff
@@ -114,11 +115,11 @@ def update_azure_firewall(cmd, instance, tags=None, zones=None, private_ranges=N
             instance.virtual_hub = SubResource(id=virtual_hub)
 
     if enable_dns_proxy is not None:
-        instance.additional_properties['DNSEnableProxy'] = enable_dns_proxy
+        instance.additional_properties['Network.DNS.EnableProxy'] = enable_dns_proxy
     if require_dns_proxy_for_network_rules is not None:
-        instance.additional_properties['DNSRequireProxyForNetworkRules'] = require_dns_proxy_for_network_rules
+        instance.additional_properties['Network.DNS.RequireProxyForNetworkRules'] = require_dns_proxy_for_network_rules
     if dns_servers is not None:
-        instance.additional_properties['DNSServer'] = dns_servers
+        instance.additional_properties['Network.DNS.Servers'] = ','.join(dns_servers or '')
```

Comment:
if `dns_servers` is `None`, should `additional_properties['Network.DNS.Servers']` equal to `None`?